libcloud/dns/drivers/rackspace.py

```diff
@@ -667,9 +667,10 @@ def _rackspace_result_has_more(response, result_length, limit):
 
 
 def _check_ptr_extra_fields(device_or_record):
-    if not (getattr(device_or_record, 'extra') and
```

Comment:
Minor thing - you can use getattr with a default value and it won't throw if the attribute is not present, it will return a default value instead (`getattr(device_or_record, 'extra', None)`).